.github/workflows/docker-base-images.yml

```diff
@@ -0,0 +1,74 @@
+name: "Docker base images"
+on:
+  # This GitHub Action will be invoked automatically from the main Azure DevOps build
+  # This job will only be launched when we are building the main branch (a snapshot) or when we are generating a release.
+  workflow_dispatch:
+    inputs:
+      azdo_build_id:
+        description: 'The specific AzDo build from which the release artifacts will be downloaded.'
+        required: true
+      is_snapshot:
+        description: 'True when we are building main branch, false when we are building a release'
+        required: true
+jobs:
+  build-and-publish-base-image:
+    runs-on: ubuntu-latest
+    env:
+      AZURE_DEVOPS_TOKEN: "${{ secrets.AZURE_DEVOPS_TOKEN }}"
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: '7.0.101'
+
+    - name: "Get current version"
+      id: versions
+      run: ./tracer/build.sh OutputCurrentVersionToGitHub
+
+    - name: "Download build assets from Azure Pipelines"
+      id: assets
+      run: ./tracer/build.sh DownloadAzurePipelineFromBuild 
+      env:
+        AzureDevopsBuildId: "${{ github.event.inputs.azdo_build_id }}"
+
+    - name: Copy tooling files to artifacts path
+      shell: bash
+      run: |
+        echo "OUTPUT ARTIFACT PATH: ${{steps.assets.outputs.artifacts_path}}"
+        cp ./tracer/build/_build/docker/reliability-env/* ${{steps.assets.outputs.artifacts_path}}
+
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v2
+
+    - name: Set up Docker Buildx
+      id: buildx
+      uses: docker/setup-buildx-action@v2
+
+    - name: Generate image tags
+      id: docker-base-image-tags
+      shell: bash
+      run: |
+        if [ "$is_snapshot" = "true" ]; then
+          echo "tag-names=ghcr.io/datadog/dd-trace-dotnet/dd-trace-dotnet:latest_snapshot" >> $GITHUB_OUTPUT
```

Comment:
I know nothing about docker push, I just notice that Zach had set tag without the `tag-names=` prefix. Do both worh?

Reply:
I didn't realize those docker registries are public... Should we be a bit more explicit on the intent of those, like having `ghcr.io/datadog/dd-trace-dotnet/dd-trace-dotnet-relenv` tag ?

Reply:
The prefix "tag-names=" is necessary in order to export value to other steps.

"dd-trace-dotnet-relenv" : Ok, I think you are right. Although this implies to change the name for java and php (dd-trace-java-relenv and dd-trace-php_relenv). I am going to make the change